tutorials/beginner/configuration.md

```diff
@@ -144,13 +144,13 @@ For a simple sitemap and rule example on the next page, we'll configure this thi
 
 ![](images/picture_21.jpg)
 
-Basically we gave the thing a name ("Wallplug TV", please note the naming convention mentioned above!) and linked the "Switch" channel.
-Now we can see the wallplug (and the Zwave controller) on the "Control" page, too. It's switch is currently turned off:
+Basically we gave the thing a name ("Wallplug TV") and linked the "Switch" channel.
+Now we can see the wallplug (and the Zwave controller) on the "Control" page, too. Its switch is currently turned off:
```

Comment:
Could you break this sentence down to a new line please?
We need that to better handle changes in the docs and it would be nice to fix this, while we are improving this page anyways. 😃